simpleclient_common/pom.xml

```diff
@@ -39,6 +39,11 @@
             <artifactId>simpleclient</artifactId>
             <version>0.0.12-SNAPSHOT</version>
         </dependency>
+        <dependency>
+            <groupId>org.yaml</groupId>
+            <artifactId>snakeyaml</artifactId>
```

Comment:
We can't add dependencies in here, it's used by the pushgateway and servlet and this may clash with other versions the user is using.